### PR TITLE
Fix: removal of encrypting variables

### PR DESCRIPTION
**Pull request by K-Codemagic**
Removed encrypting variables and added info for storing sensitive data

### CODE REVIEW

File: content/variables/environment-variable-groups.md [hunk 1]
```diff
@@ -84,14 +93,14 @@ Here you'll find some of the application environment groups and variables explai
...
+        GCLOUD_SERVICE_ACCOUNT_CREDENTIALS: # <-- Put your Google Play service account credentials here.
         FCI_KEYSTORE_PATH: /tmp/keystore.keystore
-        FCI_KEYSTORE: # <-- Put your encrypted keystore here
-        FCI_KEYSTORE_PASSWORD: # <-- Put your encrypted keystore password here
-        FCI_KEY_PASSWORD:  # <-- Put your encrypted key alias password here
-        FCI_KEY_ALIAS: # <-- Put your encrypted key alias here
+        FCI_KEYSTORE: # <-- Put your keystore here
+        FCI_KEYSTORE_PASSWORD: # <-- Put your keystore password here
+        FCI_KEY_PASSWORD:  # <-- Put your key alias password here
+        FCI_KEY_ALIAS: # <-- Put your key alias here
```
Comment (nevercode-kevin) [inline content/variables/environment-variable-groups.md:103]: I think we should change the format of this to indicate if the value needs to be base64 encoded. For example, the `FCI_KEYSTORE` needs to be encoded so we should mention it. 

Can we try making a table for these variables instead?

File: content/variables/environment-variable-groups.md [hunk 2]
```diff
@@ -103,20 +112,24 @@ The above groups contain the following variables:
...
         XCODE_SCHEME: "Runner"                
         # https://docs.codemagic.io/code-signing-yaml/signing-ios/
-        APP_STORE_CONNECT_ISSUER_ID:  # <-- Put your encrypted App Store Connect Issuer Id here 
-        APP_STORE_CONNECT_KEY_IDENTIFIER: # <-- Put your encrypted App Store Connect Key Identifier here 
-        APP_STORE_CONNECT_PRIVATE_KEY: # <-- Put your encrypted App Store Connect Private Key here 
-        CERTIFICATE_PRIVATE_KEY: # <-- Put your encrypted Certificate Private Key here 
+        APP_STORE_CONNECT_ISSUER_ID:  # <-- Put your App Store Connect Issuer Id here 
+        APP_STORE_CONNECT_KEY_IDENTIFIER: # <-- Put your App Store Connect Key Identifier here 
+        APP_STORE_CONNECT_PRIVATE_KEY: # <-- Put your App Store Connect Private Key here 
+        CERTIFICATE_PRIVATE_KEY: # <-- Put your Certificate Private Key here
```
Comment (nevercode-kevin) [inline content/variables/environment-variable-groups.md:123]: Can we make a table here instead?

File: content/variables/environment-variable-groups.md [hunk 3]
```diff
@@ -31,9 +31,14 @@ Variables defined in environment variable groups work exactly as [Environment Va
...
-Entering values in the Variable value input and marking the **Secure** checkbox will automatically encrypt those values. However, note that in order to store files as secure environment variables, the file needs to be base64 encoded first. This can be done with the help of different OS-specific command lines. To use the file, you will have to decode it during the build. 
+Entering values in the Variable value input and marking the **Secure** checkbox will automatically encrypt those values. However, note that in order to store **_binary files_** as secure environment variables, first it needs to be **_base64 encoded_** locally. To use the files, you will have to decode them during the build.
 
-Files like keystore, provisioning profiles require to be encoded in base64.
+Some commonly known binary files that need to be base64 encoded. e.g.
+- Android keystore (.jks or .keystore)
+- Provisioning profiles when manual code signing (.mobileprovision)
+- iOS distribution certificate (.p12) when manual code signing.
+
+This can be done with the help of different OS-specific command lines':
```
Comment (nevercode-kevin) [inline content/variables/environment-variable-groups.md:41]: Maybe we can add another tip here about how to see if a file is a binary file or not using `less`?

File: content/variables/environment-variable-groups.md [hunk 4]
```diff
@@ -103,20 +112,24 @@ The above groups contain the following variables:
...
-        APP_STORE_CONNECT_ISSUER_ID:  # <-- Put your encrypted App Store Connect Issuer Id here 
-        APP_STORE_CONNECT_KEY_IDENTIFIER: # <-- Put your encrypted App Store Connect Key Identifier here 
-        APP_STORE_CONNECT_PRIVATE_KEY: # <-- Put your encrypted App Store Connect Private Key here 
-        CERTIFICATE_PRIVATE_KEY: # <-- Put your encrypted Certificate Private Key here 
+        APP_STORE_CONNECT_ISSUER_ID:  # <-- Put your App Store Connect Issuer Id here 
+        APP_STORE_CONNECT_KEY_IDENTIFIER: # <-- Put your App Store Connect Key Identifier here 
+        APP_STORE_CONNECT_PRIVATE_KEY: # <-- Put your App Store Connect Private Key here 
+        CERTIFICATE_PRIVATE_KEY: # <-- Put your Certificate Private Key here 
         BUNDLE_ID: "io.codemagic.flutteryaml" # <-- Put your bundle id here
         APP_STORE_ID: 1111111111 # <-- Use the TestFlight Apple id number (An automatically generated ID assigned to your app) found under General > App Information > Apple ID.
```
Comment (nevercode-kevin) [inline content/variables/environment-variable-groups.md:125]: Can we remove explicit values and add to a table?

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

**Comment by K-Codemagic** [comment]
Changes have been made.

File: content/variables/environment-variable-groups.md [hunk 5]
```diff
@@ -53,13 +58,20 @@ For Linux machines, we recommend installing xclip:
...
+{{</notebox>}}
 
 After running these command lines, you can paste the automatically copied string into the Variable value input and check the **Secure** checkbox to store the value in encrypted form in Codemagic.
 
 Finally, base64 decode it during build time in your scripts section using the following command:
 
 `echo $YOUR_ENVIRONMENT_VARIABLE | base64 --decode > /path/to/decode/to/your_file_name.extension`
 
+{{<notebox>}}
+**Additional Tip**: When copying file contents always include any tags. e.g. Don't forget to copy `-----BEGIN PRIVATE KEY-----` and `-----END PRIVATE KEY-----` too.
```
Comment (nevercode-kevin) [inline content/variables/environment-variable-groups.md:72]: Remove the word 'Additional' from 'Additional Tip'

File: content/variables/environment-variable-groups.md [hunk 6]
```diff
@@ -80,43 +92,52 @@ Here you'll find some of the application environment groups and variables explai
...
+
+**Variable name** | **Variable value** | **Group**
+--- | --- | ---
+FCI_KEY_ALIAS | /tmp/keystore.keystore | keystore_credentials
+FCI_KEYSTORE | contents of keystore - [`base64 encoded`](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) | keystore_credentials
+FCI_KEYSTORE_PASSWORD | Put your keystore password here | keystore_credentials
+FCI_KEY_PASSWORD | Put your key alias password here | keystore_credentials
+FCI_KEY_ALIAS | Put your key alias here | keystore_credentials
+GCLOUD_SERVICE_ACCOUNT_CREDENTIALS | Put your Google Play service account credentials here | google_play_credentials
+GOOGLE_PLAY_TRACK | This must be "alpha" or above | google_play_credentials
```
Comment (nevercode-kevin) [inline content/variables/environment-variable-groups.md:109]: Not sure this is true. Please confirm with dev team!

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

**Review by nevercode-kevin — APPROVED** [review]
LGTM

**Pull request merged**